### PR TITLE
Acceptance tests

### PR DESCRIPTION
**Pull request by jcalderin**

### CODE REVIEW

File: lib/plugins/orionPlugin.js [hunk 1]
```diff
@@ -176,9 +176,9 @@ function inspectBody(req, res, callback) {
 }
 
 /**
- * Determines what is the requested action based on the request information, knowing that it is a convenience operation.
```
Comment (dmoranj) [inline lib/plugins/orionPlugin.js]: Why this change? It seems you have changed blindly headers for request....

File: lib/services/idmAuth.js [hunk 2]
```diff
@@ -23,7 +23,7 @@
 
 'use strict';
 
-var request = require('request'),
```
Comment (dmoranj) [inline lib/services/idmAuth.js]: AAAARRRGGHH! You have DEFINITELY blindly substitute "request" for "headers"

File: lib/middleware/proxy.js [hunk 3]
```diff
@@ -25,14 +25,14 @@
 
 var config = require('../../config'),
     errors = require('../errors'),
-    request = require('request'),
+    request = require('headers'),
```
Comment (dmoranj) [inline lib/middleware/proxy.js]: Look at this.

File: test/acceptance/integration/ac_mdw/steps.py [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+"""
+(c) Copyright 2014 Telefonica, I+D. Printed in Spain (Europe). All Rights
```
Comment (fgalan) [inline test/acceptance/integration/ac_mdw/steps.py]: Headers need to be fixed, as discussed by skype.

**Comment by jcalderin** [comment]
Pycharm refactorize non-python code, y close the PR since all automatic substitutions are fixed
